11.Font-Face/assets/css/style.css

```diff
@@ -0,0 +1,94 @@
+:root {
+  --container-width: 1200px;
+  --color-black: #000;
+  --color-grey: #dadada;
+}
+
+.konstruct-grey-bg {
+  display: inline-block;
+  background-color: var(--color-grey);
+}
+
+.konstruct-container {
+  width: 100%;
+  max-width: var(--container-width);
+  margin-left: auto;
+  margin-right: auto;
+}
+
+.konstruct-header {
+  min-height: 80px;
+  margin-bottom: 40px;
+  display: flex;
+  align-items: center;
+}
+
+.konstruct-header .konstruct-container {
+  display: flex;
+  align-items: center;
+}
+
+.konstruct-header__language {
+  display: block;
+  padding: 2px;
+  font: 600 22px/18px Tourney, Arial, sans-serif;
+  letter-spacing: 0.06em;
+  text-transform: uppercase;
+}
+
+.konstruct-header-nav {
+  padding: 12px 0;
+  margin: 0 50px;
+  display: flex;
+  align-items: center;
+}
+
+.konstruct-header-nav__item {
+  font: 400 22px/28px Staatliches, Arial, sans-serif;
+  letter-spacing: 0.03em;
+}
+
+.konstruct-header-nav__item--separator {
+  margin-left: 10px;
+  margin-right: 10px;
+}
+
+.konstruct-header-logo {
+  margin-left: auto;
+}
+
+.konstruct-heading {
+  display: inline-block;
+  font: 400 36px/48px 'Francois One', Arial, sans-serif;
```

Comment:
fill be beater to put this into separated options like font-family: 'Tourney', 'Arial'
to make it more readable and issue to debug